src/OnlineSales/DataAnnotations/SurrogateForeignKeyReferenceAttribute.cs

```diff
@@ -0,0 +1,11 @@
+﻿// <copyright file="SurrogateForeignKeyReferenceAttribute.cs" company="WavePoint Co. Ltd.">
```

Comment:
@gsplakmal this class needs to be deleted as we discussed in Teams.

Reply:
Deleted the file.